README.md

```diff
@@ -1,2 +1,135 @@
 # DianUserMigration-Java
-Code for mapping user data from periodic Happy Medium export to Bridge, for DIAN project
+Code for mapping user data from periodic Happy Medium export to Synapse, into the DIAN bridge project.
+
+# Project setup:
+
+Before you can run the code, your environment must have the following environmental variables set first:
+
+## Required
+**BR_EMAIL** - Your email to sign into the Arc Validation or DIAN bridge project
```

Comment:
Formatting: add line break after each parameter.  (Currently no line breaks appear when the markdown is rendered.)